astropy/io/fits/column.py

```diff
@@ -1833,7 +1833,7 @@ def info(self, attrib='all', output=None):
             and blanks.  If there are two or more attribute names, they must be
             separated by comma(s).
 
-        output : file, optional
+        output : file-like object, optional
```

Comment:
I would remove `object`. We are always expecting an object, a `Column` one or a `str` one, etc..